rasterio/rio/options.py

```diff
@@ -322,3 +322,11 @@ def bounds_handler(ctx, param, value):
     help='Use all pixels touched by features, otherwise (default) use only '
          'pixels whose center is within the polygon or that are selected by '
          'Bresenhams line algorithm')
+
+# Feature collection or feature sequence switch.
+sequence_opt = click.option(
+    '--sequence/--collection',
+    default=True,
+    help="Write a LF-delimited sequence of texts containing individual "
+         "objects (the default) or write a single JSON text containing a "
+         "feature collection object.")
```

Comment:
I've temporarily moved the sequence option back to Rasterio while we're switching the sense of it.